### PR TITLE
V2 upgrade cleanup

### PR DESCRIPTION
**Pull request by arwer13**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -3,11 +3,11 @@
...
+The Lido Ethereum Liquid Staking Protocol allows their users to earn staking rewards on the Beacon chain without locking Ether or maintaining staking infrastructure.
 
 Users can deposit Ether to the Lido smart contract and receive stETH tokens in return. The smart contract then stakes tokens with the DAO-picked node operators. Users' deposited funds are pooled by the DAO, node operators never have direct access to the users' assets.
 
-Unlike staked ether, the stETH token is free from the limitations associated with a lack of liquidity and can be transferred at any time. The stETH token balance corresponds to the amount of Beacon chain Ether that the holder could withdraw if state transitions were enabled right now in the Ethereum 2.0 network.
+Unlike staked ether, the stETH token is free from the limitations associated with a lack of liquidity and can be transferred at any time. The stETH token balance corresponds to the amount of Ether that the holder could request to withdraw.
```
Comment (TheDZhon) [inline README.md:10]: :+1: game changer literally

File: deployed-goerli.json [hunk 2]
```diff
@@ -138,12 +132,19 @@
...
   },
-  "wstethContractAddress": "0x6320cD32aA674d2898A68ec82e869385Fc5f7E2f",
-  "wstethContractConstructorArgs": [
-    "0x1643E812aE58766192Cf7D2Cf9567dF2C37e9B7F"
-  ],
-  "executionLayerRewardsVaultAddress": "0x94750381bE1AbA0504C666ee1DB118F68f0780D4",
-  "depositorPreviousAddress": "0xEd23AD3EA5Fb9d10e7371Caef1b141AD1C23A80c",
+  "wstETH": {
+    "address": "0x6320cD32aA674d2898A68ec82e869385Fc5f7E2f",
+    "deployTx": "aaaa",
```
Comment (TheDZhon) [inline deployed-goerli.json:137]: ?

File: deployed-mainnet.json [hunk 3]
```diff
@@ -335,6 +280,7 @@
       "0x"
     ]
   },
+  "previousDepositSecurityModule": "0x710B3303fB508a84F10793c1106e32bE873C24cd",
```
Comment (TheDZhon) [inline deployed-mainnet.json:283]: do we need it here?

File: deployed-mainnet.json [hunk 4]
```diff
@@ -12,12 +11,6 @@
   "miniMeTokenFactoryAddress": "0x909d05f384d0663ed4be59863815ab43b4f347ec",
   "aragonIDAddress": "0x546aa2eae2514494eeadb7bbb35243348983c59d",
   "multisigAddress": "0x55Bc991b2edF3DDb4c520B222bE4F378418ff0fA",
-  "daoTemplateDeployTx": "0xdcd4ebe028aa3663a1fe8bbc92ae8489045e29d2a6ef5284083d9be5c3fa5f19",
-  "lidoBaseDeployTx": "0xd01afeac8b9231d6d18a874c17ad3995c8702964ade5285977681e69a588f687",
-  "oracleBaseDeployTx": "0xd16e53c87b643ba68cda34e261dda7a25ebb0b492d835a03f32fa48d5ebe68cd",
-  "nodeOperatorsRegistryBaseDeployTx": "0x32a2b4ec8b59f2cdde007bbdee23ed1b75491f0ca6acc25d45bf7afbf1c8b4dd",
-  "depositorDeployTx": "0xce40c18b10854166eb69e64171cbd33ec4ba2f369623f2f438bcf684cfed0523",
-  "executionLayerRewardsVaultDeployTx": "0xd72cf25e4a5fe3677b6f9b2ae13771e02ad66f8d2419f333bb8bde3147bd4294",
```
Comment (bulbozaur) [inline deployed-mainnet.json:20]: it doesn't look good to remove deploy txs, I think it's better to add the missing ones

File: docs/dev-fork.md [hunk 5]
```diff
@@ -1,5 +1,7 @@
 ## Fork test / development
 
+**WARNING**: this doc is outdated after Lido V2 upgrade
```
Comment (TheDZhon) [inline docs/dev-fork.md:3]: :+1: this phrase is good indeed

**Review by TheDZhon — COMMENTED** [review]
Thank you for these updates, since they are pretty essential since the Lido V2 released :+1: 

Did a first pass of review :eyes:

File: package.json [hunk 6]
```diff
@@ -28,7 +28,7 @@
     "test:coverage": "yarn compile && hardhat coverage --testfiles test",
     "test:e2e": "yarn compile && ava -T 1000000 -v",
     "estimate-deposit-loop-gas": "yarn run test:unit ./estimate_deposit_loop_gas.js",
-    "compile": "hardhat compile && yarn extract-abi",
```
[the remark this answers is not in the document]
Comment (TheDZhon) [reply package.json:31]: and it does indeed

File: deployed-mainnet.json (hunk 4 above)
Comment (arwer13) [reply deployed-mainnet.json:20]: What's the point of storing them? They are not used anywhere in the code and they are easily retrievable via Etherscan?

File: SCRATCH_DEPLOY.md [hunk 7]
```diff
@@ -1,5 +1,7 @@
 # Deploy Lido protocol from scratch
 
+**NB**: at the moment deployment from scratch is broken after Lido V2 upgrade and this guide requires revision.
```
[the remark this answers is not in the document]
Comment (arwer13) [reply SCRATCH_DEPLOY.md:3]: but it's actually broken, at least due to the absence of CompositePostRebaseBeaconReceiver, old burner, change of Lido, NOR, LidoOracle contracts, etc

How about "at the moment the deployment from scratch scripts and manual target Lido V1 version (before Lido V2 upgrade) and is not working" ?

File: docs/node-operator-manual.md
[the remark this answers is not in the document]
Comment (arwer13) [reply docs/node-operator-manual.md:1]: I'm all for removing duplicating outdated versions

File: docs/oracle-operator-manual.md
[the remark this answers is not in the document]
Comment (arwer13) [reply docs/oracle-operator-manual.md:1]: removed

File: deployed-mainnet.json (hunk 3 above)
Comment (arwer13) [reply deployed-mainnet.json]: agree, removed

File: deployed-mainnet.json [hunk 8]
```diff
@@ -1,6 +1,5 @@
 {
   "deployCommit": "35d8c862406f82c64091760aec8feab0e114aac1",
```
[the remark this answers is not in the document]
Comment (arwer13) [reply deployed-mainnet.json]: Agree, its a bit better, although less accurate

File: SCRATCH_DEPLOY.md (hunk 7 above)
[the remark this answers is not in the document]
Comment (TheDZhon) [reply SCRATCH_DEPLOY.md]: :+1:

File: README.md [hunk 9]
```diff
@@ -3,11 +3,11 @@
...
 
 Before getting started with this repo, please read:
```
Comment (TheDZhon) [inline README.md:12]: Let's remove the link to Primer and change the docs link to `docs.lido.fi/contracts`, wdyt?

File: README.md
Comment (TheDZhon) [inline README.md:1]: A few suggestions for the file:
- let's change the node version to v16
- update copyright to 2020-2023
- not sure about E2E part at all

File: docs/protocol-levers.md [hunk 10]
```diff
@@ -1,5 +1,7 @@
 # Protocol levers
 
+**WARNING**: this doc is outdated after Lido V2 upgrade. For the more up-to-date version see https://docs.lido.fi/deployed-contracts/
```
Comment (TheDZhon) [inline docs/protocol-levers.md:3]: the link is wrong, I guess
should be https://docs.lido.fi/guides/protocol-levers

**Review by TheDZhon — APPROVED** [review]
LGTM except the suggested changed and deployment tx resolution discussed with @bulbozaur  :+1:

File: docs/protocol-levers.md (hunk 10 above)
Comment (arwer13) [reply docs/protocol-levers.md:3]: hmm, removed the protocol-levers.md at all and updated link in README. Does it seem better for you?

Comment (TheDZhon) [reply docs/protocol-levers.md]: yeah, ty!

File: deployed-mainnet.json (hunk 4 above)
Comment (arwer13) [reply deployed-mainnet.json:20]: Added because need them to generate artifacts.json

**Pull request merged**